Podcast/app/src/main/java/br/ufpe/cin/if710/podcast/ui/MainActivity.java

```diff
@@ -119,19 +120,30 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
         }
 
         private boolean isListDownloaded() {
-            return false;
+            SharedPreferences listSavedPref = getApplicationContext().getSharedPreferences(getString(R.string.list_saved), MODE_PRIVATE);
+            return listSavedPref.getBoolean(getString(R.string.list_saved), false);
```

Comment:
Retrieving preference which indicates the episodes have already been downloaded. _false_ is returned if it does not exists.